enterprise/frontend/src/metabase-enterprise/whitelabel/components/HelpLinkSettings/HelpLinkRadio.tsx

```diff
@@ -0,0 +1,59 @@
+import { t } from "ttag";
+import { useCallback } from "react";
+import { Radio, Stack } from "metabase/ui";
+
+interface Props {
+  setting: { value: string };
+  onChange: (value: string) => void;
+}
+
+export const HelpLinkRadio = ({ setting, onChange }: Props) => {
+  const focusUrlInput = useCallback(async () => {
+    // The url input is rendered only after the setting is saved and returned from the BE
+    // there's no easy way to autofocus it from here besides waiting.
+    // `autofocus` can't be used otherwise it will be autofocused also on page load
+    const input = await waitFor(() =>
+      document.querySelector<HTMLInputElement>(
+        "#setting-help-link-custom-destination",
+      ),
+    );
+    input?.focus();
+  }, []);
+
+  return (
+    <Radio.Group value={setting.value} onChange={onChange}>
+      <Stack>
+        <Radio label={t`Link to Metabase help`} value={"metabase_default"} />
+        <Radio label={t`Hide it`} value={"hidden"} />
+        <Radio
+          label={t`Go to a custom destination...`}
+          value={"custom"}
+          onClick={focusUrlInput}
+        />
+      </Stack>
+    </Radio.Group>
+  );
+};
+
+/**
+ * Wait for `condition` to return truthy and returns its value
+ * Checks every `interval` ms and times out after `timeout` ms
+ */
+const waitFor = <T,>(
```

Comment:
If we keep this, we should probably move it somewhere else

Reply:
as stated above this should be removed.